readthedocs/builds/models.py

```diff
@@ -320,7 +320,7 @@ def config(self):
         :rtype: dict
         """
         last_build = (
-            self.builds(manager=INTERNAL).filter(
+            self.builds.filter(
```

Comment:
We don't need to filter builds by internal/external here, if we are accessing the version, we already know if it's external o internal.